### PR TITLE
Andynuzzo/it

### PR DESCRIPTION
**Pull request by andreanuzzo**
#218 for Italian

### CODE REVIEW

**Comment by andreanuzzo** [comment]
Done

File: app/locales/it/locationTracking.json [hunk 1]
```diff
@@ -9,4 +9,5 @@
     "latest_news":"Ultime Novità",
     "url_info":"Per più informazioni, visita la homepage di Private Kit:",
     "private_kit_url":"privatekit.mit.edu"
+    "overlap": "CONTROLLA SOVRAPPOSIZIONI"
```
Comment (esantus) [inline app/locales/it/locationTracking.json]: Should it be "CONTROLLA LE INTERSEZIONI"? Check intersections in Italian sounds better... Also determiners are generally need in Italian

File: app/locales/it/overlap.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+{
+  "overlap_title": "Controlla Sovrapposizioni",
+  "overlap_para_1": "Le aree verdi sono dati aggregati della tua posizione, clicca per ingrandire. Gli indicatori rossi sono le tue posizioni registrate. \n\nClicca il pulsante per scaricare informazioni pubbliche su casi confermati, indicati come aree viola.",
```
Comment (esantus) [inline app/locales/it/overlap.json]: What is the difference between GREEN and RED? IT is not clear in this text

File: app/locales/it/overlap.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  "show_overlap": "Controllo dati pubblici",
+  "loading_public_data": "Carico dati...",
```
Comment (esantus) [inline app/locales/it/overlap.json]: Caricamento dati... Maybe this is better?

File: app/locales/it/overlap.json (hunk 2 above)
Comment (andreanuzzo) [reply app/locales/it/overlap.json]: Not clear in the English text either, tbh...

**Pull request merged**